.travis.yml

```diff
@@ -29,8 +29,7 @@ install:
         travis_retry sudo apt-get install mencoder;
       fi;
     - pip install -q coverage coveralls nose-timer nibabel flake8
-    - python setup.py build
-    - python setup.py install
+    - pip install .[save_movie]
```

Comment:
Yes, the alternative would be to add imageio to the `pip install` command, but this way we test at the same time that the optional dependency is resolved